bin/package/raspberry/files/unattended-upgrades

```diff
@@ -1,6 +1,6 @@
 Unattended-Upgrade::Origins-Pattern {
-        "origin=Debian,codename=${distro_codename},label=Debian";
-        "origin=Debian,codename=${distro_codename},label=Debian-Security";
+        "origin=Raspbian,codename=${distro_codename},label=Raspbian";
```

Comment:
💯 
just that?

Reply:
yeah, no `origin=Raspbian` packages were allowed, so no upgrades allowed as well.

Reply:
Unfortunately, it will help for only new installations, all raspberries with `origin=Debian` need manual changes.